js/tinymce/classes/dom/EventUtils.js

```diff
@@ -239,6 +238,21 @@ define("tinymce/dom/EventUtils", [], function() {
 		}
 
 		/**
+  	 * Checks if the DOM has completely loaded
+  	 *
+  	 * @method isDomLoaded
+  	 * @return {Boolean} True/false if the DOM is loaded or not
+		 */
+		self.isDomLoaded = function () {
+			if (document.readyState === "complete" || (document.readyState === "interactive" && document.body)) {
```

Comment:
Why check for `document.body`? From reading [MDN](https://developer.mozilla.org/en-US/docs/Web/API/document.readyState), shouldn't `document.body` be present when `readyState == "interactive"` ("finished parsing but still loading sub-resources") or are there cases when that's not true (I've never used `document.readyState` before)?